python/samples/concepts/memory/complex_memory.py

```diff
@@ -27,8 +27,8 @@
     WeaviateCollection,
 )
 from semantic_kernel.data import VectorStoreRecordCollection, vectorstoremodel
-from semantic_kernel.data.definitions import VectorStoreField
-from semantic_kernel.data.vectors import SearchType, VectorSearch
+from semantic_kernel.data._definitions import VectorStoreField
```

Comment:
Curious about the rationale to change it from `.definitions` -> `._definitions`? From a "public API" perspective, it feels weird to have the underscore.